cmd/foreach/foreach.go

```diff
@@ -151,6 +185,27 @@ func setupOutputFiles(campaignName string, command string) {
 	defer successfulReposFile.Close()
 	defer failedReposFile.Close()
 
+	if _, err := os.Lstat(successfulReposSymlink); err == nil {
+		err := os.Remove(successfulReposSymlink)
+		if err != nil {
+			logger.Warnf("Failed to remove previous symlink for successful repos: %v", err)
+		}
+	}
+	err := os.Symlink(successfulReposFileName, successfulReposSymlink)
+	if err != nil {
+		logger.Warnf("Failed to create symlink for successful repos: %v", err)
```

Comment:
warn or fail?
Could there be a reason why this should fail?

Reply:
good question, I didn't think we wanted the whole thing to bomb out if this went wrong. It's not really an essential piece of functionality so perhaps better if things continue working if it breaks?